tests/test-recipes/split-packages/_intradependencies/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version='1' %}
+
+package:
+  name: intradependencies
+  version: {{ version }}
+
+requirements:
+  build:
+    - python {{ python }}
+    - r-base {{ r }}
+
+outputs:
+  - name: lib1intradependencies
+    script: install-lib1intradependencies.sh
+
+  - name: py1-intradependencies
+    requirements:
+      - {{ pin_subpackage('lib1intradependencies', exact=True) }}
+      - r-base {{ r }}
+    script: install-py1-intradependencies.sh
+
+  - name: r1-intradependencies
+    requirements:
+      - {{ pin_subpackage('lib1intradependencies', exact=True) }}
+      - python {{ python }}
+    script: install-r1-intradependencies.sh
+
+  - name: lib2intradependencies
```

Comment:
I have the following layout:
```
lib1intradependencies
┃  ┃
┃  ┗ py1-intradependencies
┃
┗ r1-intradependencies

lib2intradependencies
┃  ┃
┃  ┗ py2-intradependencies
┃
┗ r2-intradependencies
```

So there is no relationship, and I ensure that files from one of these islands are never present at during the installation of a package from the other island.